lib/capistrano/tasks/deploy.rake

```diff
@@ -16,7 +16,9 @@ namespace :deploy do
       invoke 'magento:setup:di:compile'
       invoke 'magento:setup:permissions'
       within current_path do
-        execute :magento, 'maintenance:enable'
+        if test 'ls bin/magento'
```

Comment:
I'm getting ready to implement this Gem again for a new project and if I run into this issue again, I'll provide steps for how to reproduce. Standby until then.